python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -102,6 +107,18 @@ class RunningTaskInfo:
     bytes_outputs: int
 
 
+@dataclass
+class NodeMetrics:
+    num_tasks_submitted: int = field(default=0)
```

Comment:
Any reason we need to use `field` here?
```suggestion
    num_tasks_submitted: int = 0
```

Reply:
Good call, no reason, will remove / simplify them all.